src/app/api/quickbooks/auth/auth.controller.ts

```
@@ -0,0 +1,25 @@
+import authenticate from '@/app/api/core/utils/authenticate'
+import { AuthService } from '@/app/api/quickbooks/auth/auth.service'
+import httpStatus from 'http-status'
+import { NextRequest, NextResponse } from 'next/server'
+
+export async function getAuthorizationUrl(req: NextRequest) {
+  const user = await authenticate(req)
+  const token = req.nextUrl.searchParams.get('token') || ''
```

Let's panic and throw 403 if `token` is not provided